cookbook/views/api.py

```diff
@@ -864,12 +884,14 @@ def paginate_queryset(self, queryset, request, view=None):
         return super().paginate_queryset(queryset, request, view)
 
     def get_paginated_response(self, data):
-        return Response(OrderedDict([
-            ('count', self.page.paginator.count),
-            ('next', self.get_next_link()),
-            ('previous', self.get_previous_link()),
-            ('results', data),
-        ]))
+        return Response(
+            OrderedDict(
+                [
+                    ('count', self.page.paginator.count),
```

Comment:
I would have preffered if the file wasn't reformatted everywhere, maybe you can discard a few of these changes so its a bit easier to see whats actually changed.